src/aircraft/FlyByWireA380X.ts

```diff
@@ -0,0 +1,141 @@
+        },
+        [AircraftFeature.StrobeLights]: {
+            LIGHTING_STROBE_0: FeatureType.Int,
+        },
+        [AircraftFeature.TaxiLights]: {
+            LIGHTING_LANDING_1: FeatureType.Int,
+        },
+        [AircraftFeature.WingLights]: {
+            LIGHTING_WING_0: FeatureType.Int,
+        },
+        [AircraftFeature.APU]: {
+            A32NX_OVHD_APU_MASTER_SW_PB_IS_ON: FeatureType.Int,
+            A32NX_OVHD_APU_START_SW_PB_IS_ON: FeatureType.Int,
+        },
+        [AircraftFeature.Battery]: {
+            A32NX_OVHD_ELEC_BAT_1_PB_IS_AUTO: FeatureType.Int,
+            A32NX_OVHD_ELEC_BAT_2_PB_IS_AUTO: FeatureType.Int,
+            A32NX_OVHD_ELEC_BAT_ESS_PB_IS_AUTO: FeatureType.Int,
+            A32NX_OVHD_ELEC_BAT_APU_PB_IS_AUTO: FeatureType.Int,
+        },
+        [AircraftFeature.Seatbelts]: {
+            XMLVAR_SWITCH_OVHD_INTLT_SEATBELT_Position: FeatureType.Int,
+            A32NX_FLAPS_HANDLE_INDEX: FeatureType.Int,
+            A32NX_GEAR_HANDLE_POSITION: FeatureType.Int,
+        },
+        [AircraftFeature.EmergencyLights]: {
+            XMLVAR_SWITCH_OVHD_INTLT_EMEREXIT_Position: FeatureType.Int,
+        },
+        [AircraftFeature.AntiIce]: {
+            XMLVAR_Momentary_PUSH_OVHD_ANTIICE_ENG1_Pressed: FeatureType.Int,
+            XMLVAR_Momentary_PUSH_OVHD_ANTIICE_ENG2_Pressed: FeatureType.Int,
+            XMLVAR_Momentary_PUSH_OVHD_ANTIICE_ENG3_Pressed: FeatureType.Int,
+            XMLVAR_Momentary_PUSH_OVHD_ANTIICE_ENG4_Pressed: FeatureType.Int,
+            XMLVAR_Momentary_PUSH_OVHD_ANTIICE_WING_Pressed: FeatureType.Int,
+        }
+    }
+
+    flapNames: FlapNames = {
+        0: 'UP',
+        1: 'CONF 1',
+        2: 'CONF 1+F',
+        3: 'CONF 2',
+        4: 'CONF 3',
+        5: 'FULL',
+    }
+
+    match(title: string, icao: string, config_path: string): boolean {
+        if (!title.includes('a380')) {
+            return false;
+        }
+
+        return ['fbw', 'flybywire'].some((w) => title.includes(w));
+    }
+
+    beaconLights(value: number): FeatureState {
+        return value === 1;
+    }
+
+    landingLights(value: number): FeatureState {
+        return value === 1;
+    }
+
+    logoLights(value: number): FeatureState {
+        return value === 1;
+    }
+
+    navigationLights(value: number): FeatureState {
+        return value === 1;
+    }
+
+    strobeLights(value: number): FeatureState {
+        if (value === 1) {
+            return null;
+        }
+        return value === 0;
+    }
+
+    wingLights(value: number): FeatureState {
+        return value === 1;
+    }
+
+    taxiLights(value: number): FeatureState {
+        return value === 0 || value === 1;
+    }
+
+    APU(mastersw: number, startsw: number): FeatureState {
+        return mastersw === 1 && startsw === 1;
+    }
+
+    battery(bat1: number, bat2: number, ess: number, apu: number): FeatureState {
+        return bat1 === 1 || bat2 === 1 || ess === 1 || apu === 1
```

Comment:
Why checking the APU here ? We already have it separately.

Reply:
I'm checking whether the APU battery is on, but I'm unsure if I should only check BAT1 and BAT2 or also include ESS and APU BAT in the check

![image](https://github.com/user-attachments/assets/b1fb1608-87b3-4c82-b62a-2fe91d258d21)

Reply:
What does APU Battery and ESS Battery provide, this will be main question then here for A380. 

Does that systems together or separately provide enough electrical power to the systems or for example APU BAT is only used to start the APU.

If they are all required for powering up the aircraft electrically on the ground then, your implementation about checking them with `OR` is not ok, it should be `AND`. But if all of them are not required, so to say, if BAT 1 and BAT 2 is enough to power the systems then the logic should be either different or we should only check that two.

Hope it is more clear now :)

Reply:
Okay, so based on my research, the situation is that batteries 1 and 2 are used to power up the aircraft. The ESS is an emergency (backup) battery, and the APU bat is used to start the APU.

In this setup, do you think I should just check BAT 1 and 2?

Reply:
Would `(batt1 && batt2) || ess` make sense?